apps/docs/src/app/core/component-docs/table/table-docs.component.html

```diff
@@ -164,6 +164,23 @@
 
 <separator></separator>
 
+<fd-docs-section-title [id]="'with-row-unnavigatable'" [componentName]="'table'">
+    Table with Non-navigatable Row
+</fd-docs-section-title>
+<description>
+    A row can be non-navigatable by setting the property <code>[navigatable]="false"</code>, for both regular and Pop In modes.<br><br>
```

Comment:
You don't need the `, for both regular and Pop In modes` part

Reply:
Thanks I have removed it.